NETPROGRAMTest/BasicExerciseTest/Exercise08Test.cs

```diff
@@ -0,0 +1,19 @@
+﻿using NETPROGRAM.BasicExercise;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace NETPROGRAMTest.BasicExerciseTest
+{
+    [TestClass]
+    public class Exercise08Test
+    {
+
+        [TestMethod]
+        public void TestPI()
+        {
+            var test = new Exercise08();
+
+            Assert.AreEqual(test.PI(), 3.14139);
```

Comment:
```suggestion
            Assert.AreEqual(test.PI(), 3.14159);
```

Thêm delta vào 

![image](https://user-images.githubusercontent.com/10450325/78627139-ecc7f900-78ba-11ea-9d89-c4f486f5a751.png)

Reply:
done